lighthouse-core/report/v2/renderer/format-helpers.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const RATINGS = {
+  PASS: {label: 'pass', minScore: 75},
+  AVERAGE: {label: 'average', minScore: 45},
+  FAIL: {label: 'fail'}
+};
+
+/**
+ * Convert a score to a rating label.
+ * @param {number} score
+ * @return {string}
+ */
+function calculateRating(score) {
+  let rating = RATINGS.FAIL.label;
+  if (score >= RATINGS.PASS.minScore) {
+    rating = RATINGS.PASS.label;
+  } else if (score >= RATINGS.AVERAGE.minScore) {
+    rating = RATINGS.AVERAGE.label;
+  }
+  return rating;
+}
+
+/**
+ * Format number.
+ * @param {number} number
+ * @return {string}
+ */
+function formatNumber(number) {
+  return number.toLocaleString(undefined, {maximumFractionDigits: 1});
+}
+
+/**
+ * Format time.
+ * @param {string} date
+ * @return {string}
+ */
+function formatDateTime(date) {
+  const options = {
+    month: 'short', day: 'numeric', year: 'numeric',
+    hour: 'numeric', minute: 'numeric', timeZoneName: 'short'
+  };
+  let formatter = new Intl.DateTimeFormat('en-US', options);
+
+  // Force UTC if runtime timezone could not be detected.
+  // See https://github.com/GoogleChrome/lighthouse/issues/1056
+  const tz = formatter.resolvedOptions().timeZone;
+  if (!tz || tz.toLowerCase() === 'etc/unknown') {
+    options.timeZone = 'UTC';
+    formatter = new Intl.DateTimeFormat('en-US', options);
+  }
+  return formatter.format(new Date(date));
+}
+
+if (typeof module !== 'undefined' && module.exports) {
+  module.exports = {
```

Comment:
Hmm, even when I uncomment that line the compilation succeeds. Although I don't seem to get a closure-output.js file